web/server/codechecker_server/api/report_server.py

```diff
@@ -1777,6 +1779,17 @@ def _setReviewStatus(self, session, report_hash, status,
                                             review_status.date)
         session.add(system_comment)
 
+        # Confirmed, false positive and intentional reports are considered
+        # closed, so their "fix date" is set. The reports are reopened when
+        # they become unreviewed again.
+        session \
+            .query(Report) \
+            .filter(Report.bug_id == report_hash) \
+            .update({
+                "fixed_at": None if review_status.status
+                in ["unreviewed", "confirmed"]
+                else review_status.date})
```

Comment:
We need to set fix dates for reports which are already in the database. For this I think we have to create a migration script which will do this.